server/actions/Event.ts

```diff
@@ -0,0 +1,21 @@
+import mongoDB from "../index";
+import EventSchema from "../models/Event";
+import { Event } from "utils/types";
+
+/**
+ * @param id EventId string to identify an event in our database.
+ * @returns A single event.
+ */
+export const getEvent = async function (id: string) {
+    await mongoDB();
+    if (!id || id == "") {
+        throw new Error("Invalid id");
+    }
+
+    const event = (await EventSchema.findById(id)) as Event;
```

Comment:
Quick question: I know you said we don't have to use `db.collection("events")` to access the events collection in the DB because mongoose already looks for it, so how does this actually look in that "events" collection?

Reply:
Good question. Essentially Mongoose maps the schema name to a collection name. If you look at the `EventSchema` model, the schema is named `Event`. So Mongoose will look for a collection named `events` to do operations on schema `EventSchema`. https://mongoosejs.com/docs/models.html is a good reference. 

Specifically: "The first argument is the singular name of the collection your model is for. Mongoose automatically looks for the plural, lowercased version of your model name. Thus, for the example above, the model Tank is for the tanks collection in the database."